src/commands/infra/validate.decorator.json

```diff
@@ -0,0 +1,5 @@
+{
+  "command": "validate",
+  "alias": "v",
+  "description": "Validate will verify that all infrastructure deployment prerequisites have been correctly installed."
+}
```

Comment:
I am also seeing an error on the `spk infra validate` as if it doesnt exist?

```
error:   Unknown command "validate"
Usage: infra [options] [command]

Manage and modify your Bedrock infrastructure.

Options:
  -v, --verbose         Enable verbose logging
  -h, --help            output usage information

Commands:
  generate|g [options]  Generate scaffold for terraform cluster deployment.
  scaffold|s [options]  Create initial scaffolding for cluster deployment.
  generate|g [options]  Generate scaffold for terraform cluster deployment.
```

Reply:
sorry, it is a bug in my implementation.

Reply:
i am getting this, after the fix
```
./spk-macos infra validate --help
Usage: infra validate|v [options]

Validate will verify that all infrastructure deployment prerequisites have been correctly installed.

Options:
  -h, --help  output usage information
```